### DISCUSSION TITLE
LinkedIn on Profile

### DISCUSSION THREAD

**Discussion by fayasnoushad** [community/community#6601] [Other Feature Feedback, Questions, & Ideas]
Please add linkedin on profile. Because LinkedIn and GitHub is the subsidiary companies of Microsoft.

**Comment by ghost** [comment]
Hi! @FayasNoushad

1. Hope it helps, I made this mockup.
2. In this visual creation there would be the LinkedIn link in profile GitHub.
3. Hope to help GitHub and the GitHub community ;D

![image](https://user-images.githubusercontent.com/95204402/168415374-0803ca97-7ac3-4cd0-9cfa-a2207eb2dcfd.png)

**Reply by fayasnoushad** [reply]
Need a logo too

**Reply by ghost** [reply]
Hi! @FayasNoushad 

I added the logo...
![image](https://user-images.githubusercontent.com/95204402/168445381-ee971cef-9610-40b2-83a3-e05dcddcdf47.png)

**Reply by tw4** [reply]
up!

**Reply by fayasnoushad** [reply]
This is good

**Comment by tw4** [comment]
Really great idea. This feature will make our job easier and it looks stylish :D

**Comment by ghost** [comment]
Duplicate of: https://github.com/github-community/community/discussions/17278
